python/packages/agbench/src/agbench/run_cmd.py

```diff
@@ -594,8 +595,26 @@ def run_scenario_in_docker(
     autogen_repo_base = os.path.join(autogen_repo_base, "python")
     volumes[str(pathlib.Path(autogen_repo_base).absolute())] = {"bind": "/autogen_python", "mode": "rw"}
 
+    # Add the Docker socket if we are running on Linux
+    # This allows docker-out-of-docker to work, but provides access to the Docker daemon on the host.
+    # This maintains good isolation for experiment purposes (e.g., ensuring consistent initial conditions),
+    # but deminishes the security benefits of using Docker (e.g., when facing a deliberately malicious agent).
+    # since it would allow clients to mount privalaged images, volumes, etc.
+    docker_host = os.environ.get("DOCKER_HOST", "unix:///var/run/docker.sock")
```

Comment:
I think it's fine for benchmark for now. Though for deployment we should recommend Azure or docker compose